{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "children": [],
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:o:redhat:enterprise_linux:7.0:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:redhat:enterprise_linux:6.0:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:qemu:qemu:-:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:redhat:enterprise_linux:8.0:*:*:*:advanced_virtualization:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:redhat:enterprise_linux:8.0:*:*:*:-:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:redhat:enterprise_linux:9.0:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "secalert@redhat.com",
      "ID": "CVE-2024-8354"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "A flaw was found in QEMU. An assertion failure was present in the usb_ep_get() function in hw/net/core.c when trying to get the USB endpoint from a USB device. This flaw may allow a malicious unprivileged guest user to crash the QEMU process on the host and cause a denial of service condition."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://access.redhat.com/security/cve/CVE-2024-8354",
          "refsource": "",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://access.redhat.com/security/cve/CVE-2024-8354"
        },
        {
          "name": "RHBZ#2313497",
          "refsource": "",
          "tags": [
            "Issue Tracking"
          ],
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2313497"
        },
        {
          "name": "https://security.netapp.com/advisory/ntap-20241011-0008/",
          "refsource": "",
          "tags": [],
          "url": "https://security.netapp.com/advisory/ntap-20241011-0008/"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2024-11-21T09:53Z",
  "publishedDate": "2024-09-19T11:15Z"
}